Wording tweaks to DEP 0003.

carljm opened the pull request.

treyhunner commented on draft/0003-javascript-tests.rst:
```diff
@@ -88,38 +91,30 @@ Migration Path
-easy as:
+Steps to run tests from the command-line (locally or on the CI server):
 
-1. Install `Node.js`_ and `PhantomJS`_
```
I just confirmed that PhantomJS is installed locally by the Node modules we're using (I just uninstalled Phantom globally and re-ran the tests with `npm test` to confirm), so it *is* safe to remove that link.

carljm replied:
Yep, I removed it based on the same experiment.

treyhunner commented on draft/0003-javascript-tests.rst:
```diff
@@ -88,38 +91,30 @@ Migration Path
-- How should the test framework be run within the CI process?
-- How should the testing documentation be updated for the new JavaScript test
-  framework?
+1. Install `Node.js`_ and `NPM`_.
```
I'm not sure we even need to mention NPM here because it should be bundled with Node.

carljm replied:
Is it always bundled now? If so I can remove the link.

shaib replied:
FWIW, Debian still packages nodejs and npm as separate packages.

Node is 0.10.38, not sure how out-of-date that is.

treyhunner replied:
When packaged, NPM is always bundled with Node.  However, the source repositories are separate so manually installing Node from source will require installing both packages separately.  See [this related StackOverflow answer][so].  I doubt most users would be installing Node from source, so I think it would probably be safe to remove the link.

However if we do link to NPM, I think we should at least update the URL to point to the [installation instructions][npm install] in the README file.

[so]: http://stackoverflow.com/a/25232555/98187
[npm install]: https://github.com/npm/npm#super-easy-install

carljm replied:
That is pretty far out of date. However, that might be enough rationale
to reintroduce the link. (Although what we do in the docs matters a lot
more than what we do in the DEP.)

carljm replied:
I re-added the link, but pointed it to the README installation instructions instead.

treyhunner commented:
:+1: this looks good to me.

I made a note about the unnecessary link to NPM, but I don't mind keeping it in if you think it's useful.

timgraham commented:
LGTM

carljm commented:
Thanks for review! Removed link to NPM, fixed a couple other minor issues, and merged.

@treyhunner Do you prefer to post it to django-developers one more time (with links to the PRs), or do you prefer that I do? Either is fine by me.

treyhunner commented:
@carljm could you post it to django-developers please? Thank you!

I'm going to step away from Django work for a while.  I'll respond to inquiries and pick this up again within 24 hours.

shaib commented:
I had replied to comments by mail, it seems to have been eaten by a GRUE:

Debian packs NPM separately from Node, and as Trey commented, the source repositories are separate. I think the NPM link should be restored.

carljm commented:
@shaib Your comments weren't eaten by a grue, they're here; I've already restored the link :-) Thanks!

shaib commented:
@carljm thanks. I saw your response in my mail (after I added the grue comment), I still see neither my comment nor your reply on the web... but yeah, anyway, thanks.
